src/main/java/org/commcare/formplayer/db/migration/V22__init_entities_selection.java

```diff
@@ -0,0 +1,19 @@
+package org.commcare.formplayer.db.migration;
+
+import java.util.Arrays;
+
+public class V22__init_entities_selection extends BaseFormplayerMigration {
```

Comment:
@gherceg I am unsure how to best manage this migration without collision with https://github.com/dimagi/formplayer/pull/1075. There are a few challenges here - 
1.  The version number here depends on what PR will go out first. 
2. I want to put this on staging though staging already has '22' as schema version. 

It seems like best course of action here is for one of these PRs to rebase on top of the other. @gherceg Is there a possibility to depriortize roll out of Form def work so that we roll out this PR first (I think it can be 2-4 weeks before we roll this out) and then for you to base off your branch on top of this ? Or we can do the other way rounds if your changes are more closer to being rolled out. 

FYI: @snopoke

Reply:
(FYI I pulled the https://github.com/dimagi/formplayer/pull/1075 branch off of staging in order to put this PR on staging, after checking with QA that 1075 was okay to remove.)

Reply:
Thanks for flagging @shubham1g5. The shared form definition work is close to complete (final stages of QA), but I should have separated out the migration from the code anyway. Created https://github.com/dimagi/formplayer/pull/1119 to isolate the migration so we can safely merge that and avoid nasty migration conflicts.

[Ramblings] Maybe we can learn from this and do the same with this work as well? I do think having the migration and code together adds value to reviewers and likely results in better feedback, but this could be achieved by branching off the branch with the migration and adding dependent code there, and waiting for reviews before merging the migration. Probably not a bad habit to pickup despite the fact that we don't add formplayer migrations very often.

Reply:
Thanks a bunch Graham! That does solve it for us and yeah agree that if possible we should merge the migrations sooner than later.